index.html

```diff
@@ -39,7 +43,215 @@ <h1>I'm Alishah. Glad to see you!</h1>
+                <div class="snapshoot">
+                    <img src="./Image/SnapshootPortfolio(5).png" alt="">
+                </div>
+                <div class="leftBlock">
+                    <div class="primarytext1">
+                        <h3> Multi-Post
+                            <br>
+                            Stories
+                        </h3>
+                        <div class="fram1">
+                            <span class="canopy">CANOPY</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">Back End Dev</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">2015</span>
+                        </div>
+                    </div>
+                    <div class="primarytext2">
+                        <p>A daily selection of privately <br>
+                            personalized reads;no accounts or<br>
+                            sign-ups required.</p>
+                    </div>
+                    <ul class="primarytext3">
+                        <li class="lang">html</li>
+                        <li class="lang">css</li>
+                        <li class="lang JavaScript">javascript</li>
+                    </ul>
+                    <div class="primarytext4">
+                        <button class="project">See project</button>
+                    </div>
+                </div>
+            </div>
+
+        </section>
+        <section class="about-my-slef">
+            <div class="top">
+                <div class="text-container">
+                    <h2 class="first-text">About <br> Myself</h2>
+                    <p class="second-text">Hello I'm a software developer! I can help you <br>
+                        bulid a product, feature or website.<br>
+                        Look through some of my work and <br>
+                        experience! if you like what you see and <br>
+                        have a project you need coded, don't <br>
+                        hesitate to contact me.
+                    </p>
+                </div>
+
+                <div class="social-media1">
+                    <h3 class="sub-social-media ">LET’S CONNECT</h3>
+
+                    <ul class="socail-media-icon1">
+                        <li> <a href="#"><img src="./Image/Vector (1).png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Linkedin icon.png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Vector (4).png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Vector.png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Vector (3).png" alt=""></a></li>
+                    </ul>
+                </div>
+                <div>
+                    <button class="normal-button">Get my resume</button>
```

Comment:
- [ ] As specified by the previous reviewer please make sure to follow the given design for the button styling: Active, Hover, and Pressed [(see design)](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=pSU459atAyqpbx9o-0)
    - ![firefox_7KIb3eS8J1](https://user-images.githubusercontent.com/104968236/227732073-e3f10225-2758-404b-987e-d96d33081f15.png)
    - So when it's in a hover state the background color should change to `#6070FF` and when it's pressed (active state) it should change to `#2230D2`